dask/array/core.py

```diff
@@ -3090,7 +3221,8 @@ def broadcast_to(x, shape, chunks=None):
                   tuple(bd if old > 1 else (new,)
                   for bd, old, new in zip(x.chunks, x.shape, shape[ndim_new:])))
     else:
-        chunks = normalize_chunks(chunks, shape)
+        chunks = normalize_chunks(chunks, shape, dtype=x.dtype,
```

Comment:
docstring should refer to normalize_chunks for chunks=None case.

Reply:
I'm not sure I understand this comment.  What should this function say about normalize_chunks?

Reply:
Should generally point to the chunks doc-page, I imagine some possible confusion here around what rechunking might happen.